src/app/malaria-101/activity-1/activity-1-2.component.ts

```diff
@@ -1,9 +1,7 @@
-import { Component, OnInit, ViewContainerRef } from '@angular/core';
-import { Observable } from 'rxjs/Rx';
-import { DashboardService } from '../../services/dashboard.service';
-import { ToastsManager } from 'ng2-toastr/ng2-toastr';
```

Comment:
Is the ToastsManager being removed because it is not being used in this file?

Reply:
Yup!